source/reference/minio-mc/mc-license-info.rst

```diff
@@ -0,0 +1,72 @@
+===================
+``mc license info``
+===================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+.. mc:: mc license info
+
+Description
+-----------
+
+The :mc-cmd:`mc license info` command displays information about the MinIO deployment's license status.
+Specifically, whether the deployment uses the AGPLv3 Open Source license of the MinIO Commercial License.
+
+For example, for a deployment not registered with |SUBNET|, the command returns information about the AGPLv3 license similar to the following:
```

Comment:
Thought - we're introducing "SUBNET" without making a link to why that matters for licensing.

"You must register your deployment with MinIO |SUBNET| to activate your commercial license.

For example, the command returns the following information for an unregistered deployment:"

And then cap the section with

"Use mc license register to associate your deployment with your SUBNET account. If you are not already signed up for SUBNET, see the `Registration <https://min.io/pricing?jmp/docs>`__ page."

Or something.

Reply:
I think these suggestions work.